runtime/vm/resolvefield.cpp

```diff
@@ -110,18 +110,16 @@ findField(J9VMThread *vmStruct, J9Class *clazz, U_8 *fieldName, UDATA fieldNameL
 		romClass = currentClass->romClass;
 		interfaces = J9ROMCLASS_INTERFACES(romClass);
 		for (i = 0; i < romClass->interfaceCount; i++, interfaces++) {
+			/* By definition, the interfaces MUST already have been loaded as
+			 * all superclasses & interfaces are loaded before the current class
+			 */
 			J9UTF8 * interfaceName = NNSRP_PTR_GET(interfaces, J9UTF8 *);
-			J9Class* interfaceClass = internalFindClassUTF8(
-				vmStruct,
-				J9UTF8_DATA(interfaceName),
-				J9UTF8_LENGTH(interfaceName),
-				currentClass->classLoader, 
-				J9_FINDCLASS_FLAG_EXISTING_ONLY);
-
-			if (interfaceClass != NULL) { /* shouldn't be NULL, but be safe */
+			J9Class* interfaceClass = peekClassHashTable(vmStruct, currentClass->classLoader, J9UTF8_DATA(interfaceName), J9UTF8_LENGTH(interfaceName));
+
+			if (J9_EXPECTED(interfaceClass != NULL)) { /* shouldn't be NULL, but be safe */
```

Comment:
This would look better (but perform trivially worse) as Assert_VM_notNull.

Reply:
Changed to an assert